src/otx/algorithms/common/adapters/torch/amp/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Custom AMP (Automatic Mixed Precision package) in OTX."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+try:
```

Comment:
It's just a question. Is there any reason to import `XPUGradScaler` in this file?

Reply:
Just to import it with `from otx.algorithms.common.adapters.torch.amp import XPUGradScaler`.
We can remove this part and import it with `~.torch.amp.xpu_grad_scaler import XPUGRadScaler` in `xpu_optimizer_hook`.